engine/src/main/java/org/terasology/rendering/dag/AbstractNode.java

```diff
@@ -40,17 +40,20 @@
     private RenderTaskListGenerator taskListGenerator; // TODO: investigate ways to remove nodes influence on taskList
     private boolean enabled = true;
 
-    protected FBO requiresFBO(FBOConfig fboConfig, BaseFBOsManager frameBuffersManager) {
+    public void initialise(Object initialData) { } // added here to avoid adding it to all other nodes.
```

Comment:
It looks kinda odd (along with the trapped version later) but I haven't looked at the code enough yet to understand it well enough to advise better :/